website/docs/integrations/github-actions/atmos-terraform-plan.md

```diff
@@ -0,0 +1,109 @@
+
+This action is intended to be used with [Atmos Terraform Apply](/integrations/github-actions/atmos-terraform-apply)
+
+## Usage Example
+
+```yaml
+name: "atmos-terraform-plan"
+
+on:
+  workflow_dispatch:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+    branches:
+      - main
+
+# These permissions are required for GitHub to assume roles in AWS
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  plan:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Plan Atmos Component
+        uses: cloudposse/github-action-atmos-terraform-plan@v1
+        with:
+          component: "foobar"
+          stack: "plat-ue2-sandbox"
+          component-path: "components/terraform/s3-bucket"
+          terraform-plan-role: "arn:aws:iam::111111111111:role/acme-core-gbl-identity-gitops"
+          terraform-state-bucket: "acme-core-ue2-auto-gitops"
+          terraform-state-role: "arn:aws:iam::999999999999:role/acme-core-ue2-auto-gitops-gha"
+          terraform-state-table: "acme-core-ue2-auto-gitops"
+          aws-region: "us-east-2"
+
+```
+
+### Requirements
+
+This GitHub Action expects an S3 bucket, DynamoDB table, and two access roles. 
+
+#### S3 Bucket
+
+Any basic S3 Bucket can be connected to this action. For example, using Atmos stack configuration, define a bucket using the [`s3-bucket` component](https://github.com/cloudposse/terraform-aws-components/tree/main/modules/s3-bucket) with this catalog configuration:
+
+```yaml
+import:
+  - catalog/s3-bucket/defaults
+
+components:
+  terraform:
+    # S3 Bucket for storing Terraform Plans
+    gitops/s3-bucket:
+      metadata:
+        component: s3-bucket
+        inherits:
+          - s3-bucket/defaults
+      vars:
+        name: gitops-plan-storage
+        allow_encrypted_uploads_only: false
+```
+
+
+Assign this S3 Bucket ARN to the `terraform-plan-bucket` input.
+
+#### DynamoDB Table
+
+Similiarly, a basic DynamoDB table can be created with our [`dynamodb` component](https://github.com/cloudposse/terraform-aws-components/tree/main/modules/dynamodb). Set the Hash Key and Range Key as follows:
+
+```yaml
+import:
+  - catalog/dynamodb/defaults
+
+components:
+  terraform:
+    # DynamoDB table used to store metadata for Terraform Plans
+    gitops/dynamodb:
+      metadata:
+        component: dynamodb
+        inherits:
+          - dynamodb/defaults
+      vars:
+        name: gitops-plan-storage
+        # These keys (case-sensitive) are required for the cloudposse/github-action-terraform-plan-storage action
+        hash_key: id
+        range_key: createdAt
+```
+
+
+Assign this table ARN to the `terraform-plan-table` input.
+
+#### IAM Access Roles
+
+First create an access role for storing and retrieving planfiles from the S3 Bucket and DynamoDB table. Assign this role ARN to the `terraform-state-role` input.
```

Comment:
How to do this?

Reply:
added note, but link depends on https://github.com/cloudposse/terraform-aws-components/pull/735